ci/jenkins/Build.groovy.j2

```diff
@@ -208,7 +231,9 @@ stage('Build') {
       script: "${docker_run} ${ci_hexagon} ./tests/scripts/task_config_build_hexagon.sh build",
       label: 'Create Hexagon cmake config',
     )
-    make(ci_hexagon, 'build', '-j2')
+    cmake_build(ci_hexagon, 'build', '-j2')
+    make_standalone_crt(ci_hexagon, 'build')
```

Comment:
We would want to remove make_standalone_crt() here for the hexagon build.

cc: @mehrdadh